docs/conf/atlantic/2023/news/announcing-schedule.rst

```diff
@@ -0,0 +1,80 @@
+:template: {{year}}/generic.html
+
+.. post:: July 24, 2023
+   :tags: {{year}}, {{shortcode}}-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over six weeks away from the first {{city}} conference, and we're working hard to make sure it'll be as welcoming and fun as previous years.
+
+Today we're announcing the full presentation schedule.
+
+Conference platform
+--------------------
+
+The conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. 
+Our Welcome Wagon guide, to be released soon, will help it feel familiar before you've even arrived.
+Hopin will show you the conference schedule in your time zone, making it easy to follow the schedule.
+
+You will receive an invite to Hopin about one week before the conference.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`/conf/{{shortcode}}/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. 
+We're excited to continue live Q&A after most talks. The Q&A will come from your questions but moderated by our MCs.
+
+As our attendees are in various time zones, we've planned a number of equal breaks throughout the day. We are all trying to balance our screen time, so we wanted to make sure that your eyes get their rest!
+
+Instead of a separate social event, we've added time in the schedule before the start and after the end of each day for you to socialize in the hallway sessions or one on one.
```

Comment:
```suggestion
Instead of a separate social event, we've added time in the schedule before the start and after the end of each day, for you to socialize in the hallway sessions or one on one.
```